eslint-plugin-prettier.js

```diff
@@ -401,48 +402,52 @@ module.exports = {
               usePrettierrc &&
               prettier.resolveConfig &&
               prettier.resolveConfig.sync
-                ? prettier.resolveConfig.sync(context.getFilename(), {
+                ? prettier.resolveConfig.sync(sourceFileName, {
                     editorconfig: true
                   })
                 : null;
             const prettierOptions = Object.assign(
               {},
               prettierRcOptions,
               eslintPrettierOptions,
-              { filepath: context.getFilename() }
+              { filepath: sourceFileName }
             );
-
             const prettierSource = prettier.format(source, prettierOptions);
-            if (source !== prettierSource) {
-              const differences = generateDifferences(source, prettierSource);
-
-              differences.forEach(difference => {
-                switch (difference.operation) {
-                  case OPERATION_INSERT:
-                    reportInsert(
-                      context,
-                      difference.offset,
-                      difference.insertText
-                    );
-                    break;
-                  case OPERATION_DELETE:
-                    reportDelete(
-                      context,
-                      difference.offset,
-                      difference.deleteText
-                    );
-                    break;
-                  case OPERATION_REPLACE:
-                    reportReplace(
-                      context,
-                      difference.offset,
-                      difference.deleteText,
-                      difference.insertText
-                    );
-                    break;
-                }
-              });
-            }
+
+            prettier.getFileInfo(sourceFileName, {
+              ignorePath: '.prettierignore'
```

Comment:
I’m not sure if there’s some generic mechanism for exposing default config so that people can specify `eslint-plugin-prettier` to use a different file than `.prettierignore`, but given that this was broken until now I’m not sure just how much it matters… maybe one to be addressed in a later PR? 🤷‍♂️